src/core/tsi/ssl_transport_security.c

```diff
@@ -281,6 +281,22 @@ static tsi_result peer_property_from_x509_common_name(
   return result;
 }
 
+static tsi_result add_peer_pem_certificate(
+    X509 *cert, tsi_peer_property *property) {
+  BIO *bio = BIO_new(BIO_s_mem());
+  const uint8_t *contents;
+  size_t len;
+  if (!PEM_write_bio_X509(bio, cert) ||
+      !BIO_mem_contents(bio, &contents, &len)) {
```

Comment:
Looks like this makes the windows build file fails (doesn't look like it is supported on the version of openssl/boringssl used by windows).